mdf_forge/forge.py

```diff
@@ -437,6 +437,60 @@ def match_titles(self, titles):
         return self
 
 
+    def match_years(self, years=[], min = None, max = None, include=True):
+        """Add years and limits to the query.
+
+        Arguments:
+        years   (int, str or list of int or str): The years to match. Default [].
+        min     (int of str): The lower range of years to match..
+        max     (int of str): The upper range of years to match.
```

Comment:
I'm not sure what is meant by `min` and `max` being of type "int of str"; is this a typo?

Reply:
It means a user can input [2010,2011],["2010","2011"], min=2010, min="2010". It does not matter since it is all cast to strings.

Reply:
So it means "int _or_ str"? "int _of_ str" sounds like it's an integer that contains a string.

Reply:
Yes. It was suppose to be "int or str." I am changing it to "int". No more strings as an input, correct?

Reply:
No, that's not what I meant. We should _cast_ strings given as input into integers to work with internally. The user should be able to provide a string or int for a year.

Reply:
OK